assets/template/main.py

```
@@ -0,0 +1,73 @@
+    @property
+    def compressed_size(self):
+        """
+        Returns the size of the compressed dataset
+        """
+        return self.get_disk_usage("path/to/storage")
+
+    def launch(self):
+        """
+        Runs the benchmarked tool
+        """
+        pass
```

Ruff (0.12.2)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)